data-serving/scripts/setup-db/migrations/20220322114358-drop-old-confirmation-date-index.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  async up(db, client) {
+    await db.command({
+      dropIndexes: 'cases',
+      index: 'byConfirmationDateIfListed',
+    });
+  },
+
+  async down(db, client) {
+    await db.command({
+      createIndexes: 'cases',
+      indexes: indexes,
```

Comment:
`indexes` isn't defined here, I think you need to include the value from the previous migration that defined it.

Reply:
Thanks for catching that 👍